### ISSUE TITLE
2.2 - keep_query_string migration

### ISSUE THREAD

**Issue by stefandoorn** [Setono/SyliusRedirectPlugin#117]
Hi,

From version 2.2.0 the column `keep_query_string` is added to the database. The plugin doesn't provide a migration for that, though. It would be great to have migration support available, so it automatically updates like Sylius itself does.

Thanks!

Sorry - no time to PR it myself at the moment, so filing it here as a suggestion.

**Comment by loevgaard** [comment]
I am more or less against third party migrations, so that won't happen unfortunately :(

**Comment by stefandoorn** [comment]
Then maybe a comment in an upgrade / release notes?

**Comment by loevgaard** [comment]
Yes, that would be great to have, I agree! Will reopen the issue :)
